src/components/veNft/VeNftOwnedTokenCard.tsx

```diff
@@ -0,0 +1,146 @@
+
+type OwnedVeNftCardProps = {
+  token: VeNftToken
+  tokenSymbolDisplayText: string
+  hasOverflow: boolean | undefined
+}
+
+export default function OwnedVeNftCard({
+  token,
+  tokenSymbolDisplayText,
+  hasOverflow,
+}: OwnedVeNftCardProps) {
+  const [extendLockModalVisible, setExtendLockModalVisible] = useState(false)
+  const [redeemModalVisible, setRedeemModalVisible] = useState(false)
+  const [unlockModalVisible, setUnlockModalVisible] = useState(false)
+
+  const { lockAmount, lockEnd, lockDuration } = token
+  const { data: metadata } = useVeNftTokenMetadata(token.tokenUri)
+  const thumbnailUri = metadata?.thumbnailUri
+
+  const remaining = Math.max(Math.round(lockEnd - Date.now() / 1000), 0)
+
+  const { colors, radii } = useContext(ThemeContext).theme
+
+  const cardStyles: CSSProperties = {
+    display: 'flex',
+    padding: '1rem',
+    borderRadius: radii.md,
+    background: colors.background.l0,
+  }
+
+  const renderRedeemButton = () => {
+    if (hasOverflow) {
+      return (
+        <Button block onClick={() => setRedeemModalVisible(true)}>
+          <Trans>REDEEM</Trans>
+        </Button>
+      )
+    } else {
+      return (
+        <Tooltip
+          trigger={['hover']}
+          title={
+            <Trans>
+              A veNft can only be redeemed if the project currently has
+              overflow.
+            </Trans>
+          }
+        >
+          <Button block disabled>
+            <Trans>REDEEM</Trans>
+          </Button>
+        </Tooltip>
+      )
+    }
+  }
+
+  return (
+    <Card style={cardStyles}>
+      <div style={{ color: colors.text.primary }}>
+        <Row>
+          <Col span={14}>
+            <Row align="top" gutter={0}>
+              <Col span={12}>
+                <p>
+                  <Trans>Staked ${tokenSymbolDisplayText}:</Trans>
+                </p>
+                <p>
+                  <Trans>Stake duration:</Trans>
+                </p>
+                <p>
+                  <Trans>Time remaining:</Trans>
+                </p>
+              </Col>
+              <Col span={12}>
+                <p>{formattedNum(fromWad(lockAmount))}</p>
+                <p>
+                  {detailedTimeString({
+                    timeSeconds: lockDuration,
+                    fullWords: true,
+                  })}
+                </p>
+                <p>
+                  {detailedTimeString({
+                    timeSeconds: remaining,
+                  })}{' '}
+                </p>
+              </Col>
+            </Row>
+            <Row align="top" gutter={0}></Row>
+          </Col>
+          <Col span={4} />
+          <Col span={6}>
+            <Image src={thumbnailUri} alt="nft" preview={false} />
+          </Col>
+        </Row>
+        <Space direction="vertical" style={{ width: '100%' }}>
+          <Row>
+            <Button block onClick={() => setExtendLockModalVisible(true)}>
+              <Trans>EXTEND LOCK</Trans>
```

Comment:
**question**: curious the reasoning behind all-caps. We might prefer "normal" capitalization unless theres a reason to not to do it.

Reply:
Was copying from mockups, but agree it should be lower for consistency.